packages/mc-scripts/src/utils/graphql-requests.spec.ts

```diff
@@ -19,199 +23,377 @@ beforeAll(() =>
-                },
-              ],
+  describe('update custom application', () => {
+    beforeEach(() => {
+      mockServer.use(
+        graphql.mutation('UpdateCustomApplicationFromCli', (_req, res, ctx) => {
+          return res(
+            ctx.data({
+              updateCustomApplication: {
+                id: 'application-id',
+              },
+            })
+          );
+        })
+      );
+    });
+    it('should match returned data', async () => {
+      const updatedCustomAppsData = await updateCustomApplication({
+        mcApiUrl,
+        organizationId: 'organization-id',
+        applicationId: 'application-id',
+        data: {
+          url: 'https://test.com',
+          name: 'New Test name',
+          description: 'Test description',
+          entryPointUriPath: 'new-test-custom-app',
+          icon: '<svg><path fill="#000000"></path></svg>',
+          submenuLinks: [],
+          mainMenuLink: {
+            defaultLabel: 'Test',
+            labelAllLocales: [],
+            permissions: [],
+          },
+          permissions: [
+            {
+              oAuthScopes: ['view_products', 'view_customers'],
+              name: 'viewNewTestCustomApp',
             },
-          })
-        );
-      })
-    );
-  });
-  it('should match returned data', async () => {
-    const createdCustomAppsData = await createCustomApplication({
-      mcApiUrl,
-      organizationId: 'organization-id',
-      data: {
-        url: 'https://test.com',
-        name: 'New Test name',
-        description: 'Test description',
-        entryPointUriPath: 'new-test-custom-app',
-        icon: '<svg><path fill="#000000"></path></svg>',
-        submenuLinks: [],
-        mainMenuLink: {
-          defaultLabel: 'Test',
-          labelAllLocales: [],
-          permissions: [],
+            {
+              oAuthScopes: [],
+              name: 'manageNewTestCustomApp',
+            },
+          ],
         },
-        permissions: [
-          {
-            oAuthScopes: ['view_products', 'view_customers'],
-            name: 'viewNewTestCustomApp',
-          },
-          {
-            oAuthScopes: [],
-            name: 'manageNewTestCustomApp',
-          },
-        ],
-      },
-      applicationIdentifier,
+        applicationIdentifier,
+      });
+      expect(updatedCustomAppsData?.id).toEqual('application-id');
     });
-    expect(createdCustomAppsData?.id).toEqual('application-id');
   });
 });
+describe('Custom Views', () => {
+  const customViewId = 'custom-view-id';
+  const applicationIdentifier = '__local:@@custom-view-host@@';
+  describe('fetch custom view data', () => {
+    beforeEach(() => {
+      mockServer.use(
+        graphql.query('FetchCustomViewFromCli', (_req, res, ctx) => {
+          return res(
+            ctx.data({
+              organizationExtensionForCustomView: {
+                organizationId: 'organization-id',
+                customView: {
```

Comment:
Would it be ok to use the `test-data` package for the `CustomView`s mocked data?

Reply:
Thank you, that's a great suggestion, updated in ad42bb017d285a44f57a2806fa048c8a8fe8b211